contracts/EasyAuction.sol

```diff
@@ -121,7 +121,7 @@ contract EasyAuction is Ownable {
 
     uint256 public feeNumerator = 0;
     uint256 public constant FEE_DENOMINATOR = 1000;
-    uint64 public feeReceiverUserId = 0;
+    uint64 public feeReceiverUserId = 1;
```

Comment:
This is an additional change.... Should still be okay, right?

I am about to work on the tests etc...

Reply:
I see, it's needed so tat you can recover the address of the fee receiver. It would make sense that we register the owner as the receiver in the constructor, so the fees don;t go to the first user that gets registered.